actions/access_keys/search.go

```diff
@@ -51,3 +54,64 @@ func (a *Action) search(c *gin.Context) {
 
 	c.JSON(http.StatusOK, accessKeyContracts)
 }
+
+// search will fetch a list of access keys filtered by metadata
+// Search access key godoc
+// @Summary		Search access key
+// @Description	Search access key
+// @Tags		Access-key
+// @Produce		json
+// @Param		SearchAccessKeys body filter.SearchAccessKeys false "Supports targeted resource searches with filters and metadata, plus options for pagination and sorting to streamline data exploration and analysis"
```

Comment:
It shouldn't be a body, but rather a query, I wonder if it will work actually with our structures 🫤 

Also we have different now structure for handling search so, maybe for the sake of having the swagger generation you should create something similar to `filter.SearchAccessKeys` but with `SearchParams` instead of `SearchModel`

Reply:
I think it's done